src/main/resources/com/cloudbees/jenkins/support/filter/ContentFilters/config.jelly

```diff
@@ -25,8 +25,8 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
     <f:section title="Support Bundle Anonymization">
-        <f:block>
-            <f:checkbox field="enabled" checked="${it.enabled}" title="Anonymize support bundle contents"/>
-        </f:block>
+        <f:optionalBlock field="enabled" title="Anonymize support bundle contents"  checked="${it.enabled}">
```

Comment:
If I recall correctly, this means you have to check two boxes to activate / deactivate the anonymization. Could you problem a screenshot of the UI change?

Reply:
This won't change the count of the number of checkboxes, but apparently I need some more code changes for this to work. I'm going to take this commit out of the PR and put it in a separate queue.

Reply:
This won't change the number of boxes to click, and I'll provide before/after pictures. I need to also confirm that it doesn't break settings for existing installs (I thought I tested this, but I want to be a bit paranoid since I'm hitting rough edges elsewhere).

Reply:
I've switched from `<block>`/`<optionalBlock>` to `<entry>`